diff -Nur Math-Currency-0.40.orig/lib/Math/Currency.pm Math-Currency-0.40/lib/Math/Currency.pm
--- Math-Currency-0.40.orig/lib/Math/Currency.pm	2003-12-04 00:02:38.000000000 +0100
+++ Math-Currency-0.40/lib/Math/Currency.pm	2003-12-04 00:02:53.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr2/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # PROGRAM:	Math::Currency.pm	# - 04/26/00 9:10:AM
 # PURPOSE:	Perform currency calculations without floating point

diff -Nur Math-Currency-0.40.orig/t/002_basic.t Math-Currency-0.40/t/002_basic.t
--- Math-Currency-0.40.orig/t/002_basic.t	2003-12-04 00:02:38.000000000 +0100
+++ Math-Currency-0.40/t/002_basic.t	2003-12-04 00:03:11.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/perl -w
+#!/usr/bin/perl -w
 # Before `make install' is performed this script should be runnable with
 # `make test'. After `make install' it should work as `perl test.pl'
 
